launcher_scripts/nemo_launcher/core/k8s_templates/rlhf_ppo/rlhf-ppo-actor.yaml

```diff
@@ -0,0 +1,75 @@
+{{ $config := .Values.trainingConfig }}
+
+apiVersion: kubeflow.org/v1
+kind: PyTorchJob
+metadata:
+  name: {{ .Release.Name }}-actor
+  labels:
+    app: rlhf-ppo-actor
+spec:
+  pytorchReplicaSpecs:
+    Worker:
+      replicas: {{ .Values.actor.nodes }}
+      template:
+        spec:
+          containers:
+          - name: pytorch
+            image: {{ .Values.image.trainingImage }}
+            env:
+              {{- range $key, $value := $config.envVars }}
+              - name: {{ $key }}
+                value: {{ $value | quote }}
+              {{- end}}
+            {{ if eq $config.wandbKey "nil" }}
+            command: ["torchrun"]
+            args:
+              - "--nnodes={{ .Values.actor.nodes }}"
+              - "--rdzv-backend=c10d"
+              - "--rdzv-endpoint={{ .Release.Name }}-actor-worker-0"
+              - "--nproc_per_node={{ .Values.actor.numGPUs }}"
+              - "/opt/NeMo-Aligner/examples/nlp/gpt/train_gpt_ppo_actor.py"
+              - "--config-path=/config"
+              - "--config-name=config_actor.yaml"
+              - "remote_critic_rm.critic.ip={{ .Release.Name }}-critic-worker-0.{{ $config.namespace }}.svc.cluster.local"
+              - "remote_critic_rm.critic.port=5567"
+            {{ else }}
+            command: ["bash", "-c"]
+            args:
+              - "wandb login {{ $config.wandbKey }} && torchrun --nnodes={{ .Values.actor.nodes }} --rdzv-backend=c10d --rdzv-endpoint={{ .Release.Name }}-critic-0 --nproc_per_node={{ .Values.actor.numGPUs }} /opt/NeMo-Aligner/examples/nlp/gpt/train_gpt_ppo_actor.py --config-path=/config --config-name=config_actor.yaml remote_critic_rm.critic.ip={{ .Release.Name }}-critic-worker-0.{{ $config.namespace }}.svc.cluster.local remote_critic_rm.critic.port=5567"
```

Comment:
If you change to `helm install --namespace` you can rely on `{{ .Release.Namespace }}` instead of `{{ $config.namespace }}` since helm will populate the former.

Reply:
As long as helm can render {{.Release.namespace}} correctly, we just need to remove it from the config. Clear now.